app/components/sidebars/main/channels_list/filtered_list/filtered_list.js

```diff
@@ -399,6 +399,7 @@ class FilteredList extends Component {
                     keyboardDismissMode='on-drag'
                     maxToRenderPerBatch={10}
                     viewabilityConfig={VIEWABILITY_CONFIG}
+                    keyboardShouldPersistTaps='handled'
```

Comment:
you should also include `keyboardDismissMode='interactive'`

Reply:
So, change from 'on-drag' as already set to 'interactive'? I did a project-wide search for existing uses of 'keyboardDismissMode' and it seems that both 'interactive' and 'on-drag' are used in different places. Here I was thinking of setting to 'interactive' for iOS and 'on-drag' for Android as Android treats 'interactive' as 'none', but I can't seem to get the on-drag option to work on Android simulator. Is it worth detecting for OS and switching between these two options? What about all the other uses of 'keyboardDismissMode`?

Reply:
Interactive is for dragging the keyboard down with your fingers (only iOS, Android should use the hardware back button)

Reply:
Yup. Docs say that 'on-drag' is cross-platform, so that's why I was wondering ... doesn't seem to work though, so `interactive` it is. Thx.

Reply:
Done